lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -0,0 +1,424 @@
+					class="richListItem__dragAndDropIcon"
+					:class="{ [draggableIconClassName]: !!draggableIconClassName }"
+					:size="
+						size === RICH_LIST_ITEM_SIZE.SMALL
+							? ICON_SIZES.XX_SMALL
+							: ICON_SIZES.X_SMALL
+					"
+				/>
+			</div>
+			<div v-if="icon" class="richListItem__iconWrapper">
+				<ds-icon
+					:icon="icon"
+					class="richListItem__icon"
+					:size="
+						size === RICH_LIST_ITEM_SIZE.SMALL
+							? ICON_SIZES.XX_SMALL
+							: ICON_SIZES.X_SMALL
+					"
+					:class="{
+						[iconColorClass]: !!iconColor,
+					}"
+					:style="{
+						color: iconColorHex ? iconColorHex : null,
+					}"
+					@click.prevent="$emit('icon-click')"
+				/>
+			</div>
+			<div class="richListItem__content">
+				<slot name="content" />
+			</div>
+			<div v-if="$slots.meta" class="richListItem__metaData -hideOnMobile">
+				<slot name="meta" />
+			</div>
+			<div v-if="$slots.trailing" class="richListItem__trailingSlot">
+				<slot name="trailing" />
+			</div>
+		</div>
+
+		<div v-if="$slots.meta" class="richListItem__metaData -visibleOnMobile">
+			<slot name="meta" />
+		</div>
+		<div v-if="borderColor" class="richListItem__border" />
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/radiuses';
+@import '../../../../styles/settings/spacings';
+
+$rich-list-item-border-colors: (
+	default: $color-default-border,
+	inverted: $color-inverted-border,
+	primary: $color-primary-border,
+	primary-weak: $color-primary-border-weak,
+	neutral-heavy: $color-neutral-border-heavy,
+	neutral-strong: $color-neutral-border-strong,
+	neutral: $color-neutral-border,
+	neutral-weak: $color-neutral-border-weak,
+	neutral-ghost: $color-neutral-border-ghost,
+	danger: $color-danger-border,
+	danger-weak: $color-danger-border-weak,
+	fail: $color-fail-border,
+	fail-weak: $color-fail-border-weak,
+	warning: $color-warning-border,
+	warning-weak: $color-warning-border-weak,
+	success: $color-success-border,
+	success-weak: $color-success-border-weak,
+	info: $color-info-border,
+	info-weak: $color-info-border-weak,
+	accent: $color-accent-border,
+	accent-weak: $color-accent-border-weak,
+);
+
+$rich-list-item-icon-colors: (
+	default: $color-default-icon,
+	primary: $color-primary-icon,
+	primary-weak: $color-primary-icon-weak,
+	neutral: $color-neutral-icon,
+	neutral-weak: $color-neutral-icon-weak,
+	danger: $color-danger-icon,
+	fail: $color-fail-icon,
+	warning: $color-warning-icon,
+	success: $color-success-icon,
+	info: $color-info-icon,
+	accent: $color-accent-icon,
+);
+
+.richListItem {
+	$root: &;
+
+	background: $color-neutral-background;
+	cursor: default;
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	justify-content: center;
+	max-width: 100%;
+	min-height: 62px;
+	padding: $space-xxxxs $space-xxxxs $space-xxxxs 0;
```

Comment:
Where is this padding coming from? I understand 4px left for border but rest of it?

Reply:
Ok. It is for mobile version. But on figma I see it is 4px from all sides.